modules/amf-registration/amf-registration-service/service.xml

```diff
@@ -29,11 +29,11 @@
 		<column name="emailAddress" type="String" />
 		<column name="userName_" type="String" />
 		<column name="gender" type="String" />
-		<column name="birthday" type="Date" />
+		<column name="birthday" type="String" />
 		<column name="password1" type="String" />
 		<column name="password2" type="String" />
-		<column name="homePhone" type="int" />
-		<column name="mobilePhone" type="int" />
+		<column name="homePhone" type="String" />
+		<column name="mobilePhone" type="String" />
```

Comment:
you can use `Data` instead of `String` for database storage. when implementing your logic, you can use a `parse` function to convert it.